### PR TITLE
doc: add use sealer image offline

### PR DESCRIPTION
**Pull request by YTGhost**
### Does this pull request fix one issue?

Fixes https://github.com/sealerio/sealer/issues/2001

### CODE REVIEW

File: src/docs/advanced/use-sealer-image-offline.md [hunk 1]
```diff
@@ -0,0 +1,496 @@
...
+
+It's common that some k8s clusters have their own private image registry, and they don't want to pull images from other registry for some reasons. This page is about how to integrate kyverno into k8s cluster, which will redirect image pull request to Specified registry.
+
+## How to use Kyverno BaseImage
+
+We provide official BaseImage which integrates kyverno into cluster.
+
+| Sealer version |                        Image                         | Arch  |                              OS                              | Network plugins |     Container runtime     |
+| :------------: | :--------------------------------------------------: | :---: | :----------------------------------------------------------: | :-------------: | :-----------------------: |
+|     v0.9.1     |    docker.io/sealerio/kubernetes-kyverno:v1.18.3     |  x86  | CentOS/RHEL 7.5<br/>CentOS/RHEL 7.6<br/>CentOS/RHEL 7.7<br/>CentOS/RHEL 7.8<br/>CentOS/RHEL 7.9 |     calico      | Official docker v19.03.15 |
```
Comment (Stevent-fei) [inline src/docs/advanced/use-sealer-image-offline.md:13]: modified to this？
docker.io/sealerio/kubernetes-kyverno:v1-18-3-sealerio-1

The following modifications are the same as above. cc@Starnop

Comment (Starnop) [reply src/docs/advanced/use-sealer-image-offline.md:13]: aggre with you

Comment (YTGhost) [reply src/docs/advanced/use-sealer-image-offline.md]: Fixed

File: src/docs/advanced/use-sealer-image-offline.md [hunk 2]
```diff
@@ -0,0 +1,496 @@
...
+
+`sealer run localhost/kubernetes-kyverno-nginx:v1 -m 10.0.0.238 -n 10.0.0.239 10.0.0.240 -p Sealer.io@123`
+
+## How to use Kyverno AppImage
+
+We also provide official Kyverno AppImage, you can use Kyverno AppImage when your k8s cluster don't integrate with kyverno.
+
+| Sealer version |                         Image                         |
+| :------------: | :---------------------------------------------------: |
+|     v0.9.1     | docker.io/sealer-io/scratch-kyverno:v1-6-3-sealerio-1 |
```
Comment (Stevent-fei) [inline src/docs/advanced/use-sealer-image-offline.md:78]: sealer-io -> sealerio

Comment (YTGhost) [reply src/docs/advanced/use-sealer-image-offline.md]: Fixed

**Review by kakaZhou719 — APPROVED** [review]
LGTM，@YTGhost pls fix code conflicts.

**Comment by YTGhost** [comment]
> LGTM，@YTGhost pls fix code conflicts.

Fixed

**Pull request merged**